scripts/install.sh

```diff
@@ -90,6 +91,11 @@ for arg; do
     VERIFY_SIGNATURE=0
     echo "Disabling signature verification, this is not recommended"
   fi
+
+  if [ "$arg" = "--verify-signature" ]; then
```

Comment:
Do we have these flags in our docs anywhere? It might be helpful to explain why we have both `--no-verify-signature` and `--verify-signature` flags. Typically, I'd also recommend supporting a `-h`/`--help` flag but I probably wouldn't think to try that on an install one-liner.

Reply:
We traditionally haven't been great about documenting our CLI flags, but it's on our roadmap. Will add to our list of flags to ducment/